### PR TITLE
Add volunteer cta on conf landing page, Add Ana Medina's talk

### LINKED ISSUE #78: Add volunteer interest form to the conference website
### Description

We need a way for volunteers to sign up for the 2019 conference

**Why this is valuable:**

Provides reach to the community

### Acceptance Criteria

- [ ] information about volunteering
- [ ] way to sign up to volunteer

### Resources
- the form is a google form

### PR DESCRIPTION
**Pull request by deweydell**
### Proposed changes

- Add CTA to conference page
- Embed google form on volunteers page

### Acceptance criteria

- [X] information about volunteering
- [X] way to sign up to volunteer
- [x] Fulfilled Acceptance Criteria

### Requested feedback

Give feedback on cta on https://deploy-preview-79--writespeakcode.netlify.com/2019/ and the volunteer page https://deploy-preview-79--writespeakcode.netlify.com/2019/volunteer

### CODE REVIEW

**Review by coilysiren — DISMISSED** [review]
So you're not left hanging @deweydell: I don't have the context required to meaningfully review this, so I'm gonna 👍 to acknowledge that its happening

**Comment by deweydell** [comment]
thanks @lynncyrin ❗️just waiting for the green light to release the call for volunteers, I think it's next week.

**Pull request merged**
